en/assembly/quick_start_pixhawk4.md

```diff
@@ -1,22 +1,26 @@
 # Pixhawk 4 Wiring Quick Start

```

Comment:
@barzanisar FYI, most of the change on this doc was improving the English and fixing up some cross links. It is pretty good, except for the Power section, which I think needs a LOT of work.